stablehlo/transforms/CMakeLists.txt

```diff
@@ -16,6 +16,10 @@ set(LLVM_TARGET_DEFINITIONS Passes.td)
 mlir_tablegen(Passes.h.inc -gen-pass-decls)
 add_public_tablegen_target(PassesIncGen)
 
+set(LLVM_TARGET_DEFINITIONS ChloDecompositionPatterns.td)
```

Comment:
This will work after renaming `CHLODecompositionPatterns.td` to ChloDecompositionPatterns.td`, but for now doesn't build on my machine.